### PR TITLE
Add Readme

### PR DESCRIPTION
**Pull request by jstriebel**
Adds a Readme with minimal information about the spec, it's current state, and how to work with it.

### CODE REVIEW

**Comment by jstriebel** [comment]
cc @joshmoore @MSanKeys963 @alimanfoo

File: README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Zarr Specs
+
+![](https://raw.githubusercontent.com/zarr-developers/community/main/logos/logo2.png)
```
Comment (joshmoore) [inline README.md:3]: FYI: https://github.com/zarr-developers/community/blob/main/logos/README.md

**Comment by joshmoore** [comment]
Generaly, :100:. I'd get this in ASAP and we can iterate on it as we go.

File: README.md (hunk 1 above)
Comment (jstriebel) [reply README.md:3]: Thanks, good point. I copied that link from the zarr-python Readme, I'll add the change there, too.
```suggestion
![](https://github.com/zarr-developers/zarr-logo/releases/download/2022-04-28/zarr-pink-horizontal-transparent.png)
```

Comment (jstriebel) [reply README.md]: PS: The `horizontal` pngs contain a lot of whitespace on the top and bottom and look weird, when included in the Readme directly. Therefore I used the stacked version for now.

**Comment by jstriebel** [comment]
@joshmoore @MSanKeys963 Is there anything missing to merge this?

**Comment by MSanKeys963** [comment]
Apologies for missing this, @jstriebel.

I think we're good to go here, @joshmoore. Let's merge this.

**Pull request merged**